src/handlers/motions/motionCreated/helpers.ts

```diff
@@ -14,22 +13,25 @@ import {
 
 export const getParsedActionFromMotion = async (
   motionId: string,
-  colonyClient: AnyColonyClient,
+  colonyAddress: string,
+  clients: NetworkClients[],
```

Comment:
Would it make sense to restrict this type to only the clients that can parse actions? I think this currently also includes network and token clients